gradle.properties

```diff
@@ -1 +1 @@
-org.gradle.jvmargs=-Xmx1g
+org.gradle.jvmargs=-Xmx2g
```

Comment:
Is this needed for all platforms? [Linux CI already has this](https://github.com/wpilibsuite/allwpilib/blob/master/.github/workflows/ci.yml#L22).

Reply:
Ah. I didn't realize that. I'll try without this change.